.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: E2E on Chrome
```

Comment:
:bangbang: Change the name to simply "Tests" (we're not specifically using Chrome here)